wagtailnhsukfrontend/templates/wagtailnhsukfrontend/expander.html

```diff
@@ -0,0 +1,2 @@
+
+{% include './details.html' %}
```

Comment:
Instead of altering the block context with `get_context`, you could do
```
{% include './details.html' with expander=True %}
```
To turn the details template into an expander